web/www/horas/Help/credits.html

```diff
@@ -176,6 +176,11 @@ <H1 ALIGN CENTER>Credits</H1>
 
 <P ALIGN=CENTER>+++++</P>
 
+<B>Spanish</B> translation is still under development. The Spanish Vulgate translation used is the <A HREF="http://lasagradabiblia.jimdo.com/">Petisto-Torres Amat Sacred Bible</A> (La Sagrada Biblia Petisto-Torres Amat)
+For the missal translation, the <I>Misal diario latino-español y devocionario</I> (21st edition, 1962) has been used.
```

Comment:
I've been checking the law and I'm afraid the book is not under public domain until 2036. However I've found a new ordinary of the mass (http://www.unavocesevilla.com/wp-content/uploads/2014/12/CUADERNILLO-ORDINARIO-MISA-TRADICIONAL-UNA-VOCE-SEVILLA-2015.pdf) and the Proprium de Tempore (http://rinconliturgico.blogspot.com.es/p/propios-pdf.html). I have send them both an email and hopefully they will allow me to use their translation.